model_selection_demo.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3d8b9b58",
```

Comment:
Let's change name of this notebook to get rid of "Tutorial" and "This tutorial will" phrase at the start.
Maybe also change the name of notebook to: `hyperparameter_optimization.ipynb`?

Reply:
Be careful though that links in Readme are properly updated if you change name of any notebook/file